src/backend/utilities/reportBalance.ts

```diff
@@ -46,7 +47,8 @@ export function reportBalance() {
   setInterval(async () => {
     try {
       const { identity } = await keypairsPromise;
-      const balances = await Balance.getBalances(identity.address);
+      const api = ConfigService.get('api');
+      const balances = (await api.query.system.account(identity.address)).data;
```

Comment:
This is somewhat less clear than `getBalances`. Is this the tradeoff to removing some API wrappers from the SDK?

Reply:
yes, exactly. handling balances is such a common operation in the substrate ecosystem that it is kind of strange that the kilt sdk wraps this